### ISSUE TITLE
Funny things happen when releasing

### ISSUE THREAD

**Issue by matikbird** [szwacz/electron-boilerplate#191]
Hi guys,
I'm having a hard time trying to make it work all the modules in the app.

Here's the deal:
When I run **npm start**, everything works fine. A small error o advice here and there, but no big deal.
When I run **npm release**, the **_Export PDF Option_** and a extension stops working.

You can reproduce it here:
https://github.com/matikbird/textiled

I'm using medium-editor & electron-html-to as dependences.
Medium-Editor works without the insert-plugin 3.0 extension if I release it (which I did, without the export pdf and some other options).

**Comment by black-snow** [comment]
Which version of the boilerplate are you using?

**Comment by matikbird** [comment]
3.0 I think, the one you've released like three days ago.
you can reproduce it there

**Comment by szwacz** [comment]
@matikbird please be more specyfic, what exactyly stops working, what is the error message?
If it's easy to reproduce in you rapp please specify the exact sequence of clicks :)

**Comment by matikbird** [comment]
Well, we got no error message, since I found no way to see any errors in the released version. **These problems only happens on the released version.**

Check it out, you open the app with `npm start` after the whole thing is installed.
![screenshot_2016-07-29_21-40-56](https://cloud.githubusercontent.com/assets/12424545/17266934/274c23d8-55d5-11e6-8b8f-0337f2f155b6.png)

Then you write whatever you want. Now, hit the **Article as PDF** button. It should be stored somewhere in a certain path (which you can set in the preferences section). It works. So far so good.

Now, try the same when you run **npm run release**. In the binary, it does not work. And some other things do not work also, but we get that breach when we get to it. First, let's find out why some modules like **electron-to-html** is not working in the binarie. It should export the PDF as expected.

Thanks for the fast reply, @szwacz. And all your work, of course. It's really neat.

**Comment by black-snow** [comment]
@matikbird as a last resort you can always `console.log()` or `console.error()`. Then start the app via command prompt to see the outputs.

**Comment by szwacz** [comment]
there is nothing what prevents you from opening devtools in production app other than this line: https://github.com/szwacz/electron-boilerplate/blob/master/app/background.js#L35 :)

**Comment by matikbird** [comment]
Will try in a few days, @szwacz 
I thought the dev-tools was removed from the app when releasing the production version.

I'm developing an Android & iOS app for the same repo. I uploaded everything so others can help me and it can be like Linux, a software of everyone.

I'd assume that something is not copying right when run `npm run release`, but that can be a wrong guess.
I get back to you as soon as I can.

Thanks @black-snow @szwacz for the kind support (:
